pkg/storagev2/transaction.go

```diff
@@ -109,6 +109,10 @@ func (tx *TxState) Done() {
 	})
 }
 
+func NewChildTxState(tx *TxState) *TxState {
```

Comment:
Some description would be nice. Also, I'd recommend moving it below the `NewTxState` constructor.